### DISCUSSION TITLE
buddhism/latest/heart_sutra_memorization/

### DISCUSSION THREAD

**Discussion by giscus** [shane0/buddhism#115] [General]
# buddhism/latest/heart_sutra_memorization/

https://shanenull.com/buddhism/latest/heart_sutra_memorization/

**Comment by shane0** [comment]
# heart sutra resources

- this page shows the first letter of each word, click show line to reveal the next line
- sutra study: https://ctworld.org/Buddhist%20e-Books/Books05/index.html
- this is also a command line app https://github.com/shane0/buddhism/blob/main/plugins/h.py
- screencast https://youtu.be/vllKiMq-q3M

**Reply by shane0** [reply]
# bdk

- [commentary on heart sutra bdk](https://github.com/shane0/buddhism/issues/113)
